.ci/benchmarks.groovy

```diff
@@ -109,3 +125,47 @@ def withESBenchmarkEnv(Closure body) {
     body()
   }
 }
+
+def downloadPNGReport() {
+  withSecretVault(
+      secret: "${KIBANA_SECRET}", 
+      data: ['username': 'KIBANA_USER', 
+            'password': 'KIBANA_PASSWORD', 
+            'endpoint': 'KIBANA_ENDPOINT'] ) {
+    sh(label: 'Run unit tests', script: ".ci/scripts/download-png-from-kibana.sh $KIBANA_ENDPOINT $KIBANA_USER $KIBANA_PASSWORD ${env.PNG_REPORT_FILE}")
+    archiveArtifacts(
+      allowEmptyArchive: false,
+      artifacts: "${env.PNG_REPORT_FILE}",
+      defaultExcludes: false
+    )
+  }
+}
+
+def sendSlackReportSuccessMessage() {  
+  slackMessageBlocks = [
+    [
+      "type": "section",
+      "text": [
+        "type": "mrkdwn",
+        "text": "Nightly benchmarks succesfully executed\n\n [Jenkins Build ${env.BUILD_DISPLAY_NAME}](${env.BUILD_URL})"
+      ]
+    ],
+    [
+		  "type": "divider"
+	  ],
+    [
+      "type": "section",
+      "text": [
+        "type": "mrkdwn",
+        "text": "[Kibana Dashboard](${env.KIBANA_DASHBOARD_URL})"
+      ],
+      "accessory": [
+        "type": "image",
+        "image_url": "${env.BUILD_URL}artifact/${env.PNG_REPORT_FILE}",
+        "alt_text": "No image for now"
+      ]
+    ]
+  ]
+
+  slackSend(channel: ${env.SLACK_CHANNEL}, blocks: slackMessageBlocks)  
```

Comment:
interpolation is required when using `${env.}` or use directly the env map;

```suggestion
  slackSend(channel: "${env.SLACK_CHANNEL}", blocks: slackMessageBlocks)  
```

or 

```suggestion
  slackSend(channel: env.SLACK_CHANNEL, blocks: slackMessageBlocks)  
```